orchestrator/careplanservice/handle_get.go

```diff
@@ -0,0 +1,191 @@
+package careplanservice
+
+import (
+	"context"
+	"encoding/json"
+	fhirclient "github.com/SanteonNL/go-fhir-client"
+	"github.com/SanteonNL/orca/orchestrator/lib/auth"
+	"github.com/SanteonNL/orca/orchestrator/lib/coolfhir"
+	"github.com/rs/zerolog/log"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+	"net/http"
+	"strings"
+)
+
+func (s *Service) handleGet(httpResponse http.ResponseWriter, request *http.Request) error {
```

Comment:
I'd split this in handler functions, just like we do for the write operations? Why diverge?

Reply:
An early draught just to get the flow working, these methods will be split into their own handlers in the next revision